doc/articles/intro.md

```diff
@@ -57,11 +57,64 @@ Browse the set of available controls and their properties.
 Uno Platform lets you write an application once in XAML and C#, and deploy it to any target platform. 
 
 <br/>
+<br/>
+
+***
+
+
+## Top questions about Uno Platform
+
+#### What platforms can I target with Uno Platform?
+
+Uno Platform applications run on Web (via WebAssembly), Windows, Linux, macOS, iOS and Android. [Check supported platform versions.](getting-started/requirements.md)
+
+#### Are Uno Platform applications native?
+
+Yes - Uno Platform taps into the native UI frameworks on most supported platforms, so your final product is a native app. [Read more about how Uno works.](what-is-uno.md)
+
+#### Can applications look the same on all platforms?
+
+Yes. Unless you specify otherwise, your application's UI renders exactly the same on all targeted platforms, to the pixel. Uno achieves this by taking low-level control of the native visual primitives on the targeted platform. [Read more about how Uno works.](what-is-uno.md)
+
+#### How is Uno Platform different from Xamarin.Forms or .NET MAUI?
+
+At a very high level, Uno Platform and Xamarin.Forms/.NET MAUI have different starting points and tap different ecosystems.  We believe WinUI as a starting point has the richest API set and styling engine which can be used to create pixel-perfect applications on all Web, Desktop and Mobile platforms. Xamarin.Forms and its successor .NET MAUI stem from ‘mobile origins’ and aim to bring a shared skills and tooling ecosystem to all platforms Xamarin.Forms currently supports, and more platforms in the future.
```

Comment:
> We believe

Everything else is written at the 3rd person and this is first person.

Reply:
Fair, but I think there's a good reason - everything other point is fairly objective, and this is one is more subjective ;)